docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -0,0 +1,311 @@
+    logging.getLogger("py4j.py4j.clientserver").setLevel(logging.ERROR)
+
+    configure_project(package_name)
+    with KedroSession.create(env=env, conf_source=conf_source) as session:
+        session.run()
+
+
+if __name__ == "__main__":
+    main()
+```
+
+2. **Define a new entry point**: Open `<project_root>/src/setup.py` in a text editor or IDE and add a new line in the definition of the `entry_point` tuple, so that it becomes:
+
+```python
+entry_point = (..., "databricks_run = <package_name>.databricks_run:main")
+```
+
+Remember to replace <package_name> with the correct package name for your project.
+
+This process adds an entry point to your project which can be used to run it on Databricks.
+
+```{note}
+Because you are no longer using the default entry-point for Kedro, you will not be able to run your project with the options it usually provides. Instead, the `databricks_run` entry point in the above code and in the Databricks Iris starter contains a simple implementation of two options:
+- `--package_name` (required): the package name (defined in `setup.py`) of your packaged project.
+- `--env`: specifies a [Kedro configuration environment](../../configuration/configuration_basics.md#configuration-environments) to load for your run.
+- `--conf-source`: specifies the location of the `conf/` directory to use with your Kedro project.
+```
+
+### Plan your project's output
+
+You will not be able to directly see your project's log output when it is run as a Databricks job. You will need to decide on and implement a strategy for storing your project's results. Some examples are:
+
+- Write the result of a data engineering pipeline to a managed delta table.
+- Upload metrics from a machine learning model to MLflow.
+- Save a dataset as a CSV file in DBFS.
+
+In this guide, your project will save its logs to a location in DBFS (`/dbfs/FileStore/iris-databricks/info.log`).
+
+**The `databricks-iris` starter is already configured to save logs in this location.** The configuration used to achieve this can be found in the `conf/base/logging.yml` file.
+
+### Package your project
+
+To package your Kedro project for deployment on Databricks, you must create a Wheel (`.whl`) file, which is a binary distribution of your project. In the root directory of your Kedro project, run the following command:
+
+```bash
+kedro package
+```
+
+This command generates a `.whl` file in the `dist` directory within your project's root directory.
+
+### Upload project data and configuration to DBFS
+
+```{note}
+A Kedro project's configuration and data do not get included when it is packaged. They must be stored somewhere accessible to allow your packaged project to run.
+```
+
+Your packaged Kedro project needs access to data and configuration in order to run. Therefore, you will need to upload your project's data and configuration to a location accessible to Databricks. In this guide, we will store the data on the Databricks File System (DBFS).
```

Comment:
The note seems redundant given the paragraph right below?

```suggestion
Your packaged Kedro project needs access to data and configuration in order to run. Therefore, you will need to upload your project's data and configuration to a location accessible to Databricks. In this guide, we will store the data on the Databricks File System (DBFS).
```

Reply:
I think it is redundant, but the contents in the note might not be totally clear to an inexperienced user. They may assume that their data and config got packaged and that this is an optional step. I want to avoid this if I can by making it explicit. That being said, I'm only borderline about keeping the note, if you don't think this reasoning makes sense we can remove it.